include/core/truthTable/truth_table.hpp

```diff
@@ -17,6 +17,8 @@
 
 namespace syrec {
 
+    auto transformCharToCubeValue(const char& c) -> std::optional<bool>;
```

Comment:
I'd propose to make this function a static member function of the `Cube`class in order to avoid it being a free floating function. Something like
```c++
static auto getValue(const char& c) -> Value {
  ...
}
```

Reply:
done 👍🏼